<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>352</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Drawing App</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionCircle"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionLine"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuArrange">
    <property name="title">
     <string>Arrange</string>
    </property>
    <addaction name="actionBring_To_Front"/>
    <addaction name="actionSend_to_Back"/>
    <addaction name="actionBring_Forward"/>
    <addaction name="actionSend_Backward"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_me_3"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuArrange"/>
   <addaction name="menuDraw"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectionTool"/>
   <addaction name="actionDrawCircle"/>
   <addaction name="actionDrawRectangle"/>
   <addaction name="actionDrawLine"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="VEProp"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/CircleIcon.png</normaloff>:/Icons/resource/CircleIcon.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="statusTip">
    <string>Draw a Circle</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/RectangleIcon.png</normaloff>:/Icons/resource/RectangleIcon.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/LineIcon.png</normaloff>:/Icons/resource/LineIcon.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionDrawCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/CircleIcon.png</normaloff>:/Icons/resource/CircleIcon.png</iconset>
   </property>
   <property name="text">
    <string>DrawCircle</string>
   </property>
   <property name="toolTip">
    <string>Draw Circle</string>
   </property>
   <property name="statusTip">
    <string>Click-drag to begin drawing; click on canvas to show Circle Draw Dialog</string>
   </property>
  </action>
  <action name="actionSelectionTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/CursorIcon.png</normaloff>:/Icons/resource/CursorIcon.png</iconset>
   </property>
   <property name="text">
    <string>SelectionTool</string>
   </property>
   <property name="toolTip">
    <string>Selection Tool</string>
   </property>
   <property name="statusTip">
    <string>Click to select; Shift-Select for multiple select</string>
   </property>
  </action>
  <action name="actionDrawRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/RectangleIcon.png</normaloff>:/Icons/resource/RectangleIcon.png</iconset>
   </property>
   <property name="text">
    <string>DrawRectangle</string>
   </property>
   <property name="statusTip">
    <string>Click-drag to begin drawing; click on canvas to show Rectangle Draw Dialog</string>
   </property>
  </action>
  <action name="actionDrawLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/LineIcon.png</normaloff>:/Icons/resource/LineIcon.png</iconset>
   </property>
   <property name="text">
    <string>actionDrawLine</string>
   </property>
   <property name="toolTip">
    <string>Draw Line</string>
   </property>
   <property name="statusTip">
    <string>Click to draw 1st point, click again to finish it.</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/undo.png</normaloff>:/Icons/resource/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/redo.png</normaloff>:/Icons/resource/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/trash_can.png</normaloff>:/Icons/resource/trash_can.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionBring_To_Front">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/bringToFront.png</normaloff>:/Icons/resource/bringToFront.png</iconset>
   </property>
   <property name="text">
    <string>Bring to Front</string>
   </property>
  </action>
  <action name="actionSend_to_Back">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/sendToBack.png</normaloff>:/Icons/resource/sendToBack.png</iconset>
   </property>
   <property name="text">
    <string>Send to Back</string>
   </property>
  </action>
  <action name="actionBring_Forward">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/bringForward.png</normaloff>:/Icons/resource/bringForward.png</iconset>
   </property>
   <property name="text">
    <string>Bring Forward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+}</string>
   </property>
  </action>
  <action name="actionSend_Backward">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/sendBackward.png</normaloff>:/Icons/resource/sendBackward.png</iconset>
   </property>
   <property name="text">
    <string>Send Backward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+{</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/open.png</normaloff>:/Icons/resource/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/save.png</normaloff>:/Icons/resource/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/resource/new.png</normaloff>:/Icons/resource/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbout_me_3">
   <property name="text">
    <string>About me :3</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
